### PR TITLE
Add Icon for TV Launcher

### PR DESCRIPTION
**Pull request by david-allison-1**
## For Review

⚠️  Confirm that this will not submit us to the TV App Store - we're not (and probably never will be) ready. the docs seem ambiguous. It seems to be a Play Console Checkbox - but need to make sure.

## Purpose / Description
Was requested 

## Fixes
Related: #1643 

## How Has This Been Tested?

It's not great - we can't resize bitmaps until API 23
image

## Learning (optional, can help others)
https://developer.android.com/training/tv/start/start#tv-activity

## Checklist
- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code

### CODE REVIEW

**Comment by mikehardy** [comment]
According to their document (which is really good / thorough!?) https://developer.android.com/distribute/best-practices/launch/distribute-tv - you have to opt-in, so making this better on TV will not auto-submit it, but is of course a requirement to opting in

File: AnkiDroid/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -62,7 +65,7 @@
         android:usesCleartextTraffic="true"
         android:largeHeap="true"
         android:networkSecurityConfig="@xml/network_security_config"
-        >
+        android:banner="@drawable/tv_banner">
```
Comment (mikehardy) [inline AnkiDroid/src/main/AndroidManifest.xml:68]:
```suggestion
        android:banner="@drawable/tv_banner"
        >
```

Maintain Arthur-compatible XML formatting for smaller line diffs in XML ;-)

**Review by mikehardy — APPROVED** [review]
I won't consider a white-space comment reason not to approve but would be good to alter prior to merge. When you're happy merge away, I like it!

**Comment by david-allison-1** [comment]
Requesting squash-merge

**Comment by david-allison-1** [comment]
> According to their document (which is really good / thorough!?) https://developer.android.com/distribute/best-practices/launch/distribute-tv - you have to opt-in, so making this better on TV will not auto-submit it, but is of course a requirement to opting in

The TV documentation is /really/ good, but we don't have the bandwidth to properly implement leanback support and recode the app. There's also the question of whether the benefit will be worth the effort, as it's a massive undertaking for an app which likely wouldn't see that much usage as it's not a typical TV app.

**Comment by mikehardy** [comment]
Yeah - I wouldn't advocate actually *doing it* for that same reason - likely very niche audience - unless someone was super motivated, and I am not :-). But I was surprised by the docs, they were good. EIther way, no worry about accidentally opting in

**Pull request merged**